contributing_to_docs/tools_and_setup.adoc

```diff
@@ -116,3 +116,54 @@ edit existing content or create assemblies and modules.
 some basic guidelines to keep things consistent across our content.
 * link:create_or_edit_content.adoc[Create a local working branch] on your
 workstation to edit existing content or create content.
+
+=== How to deploy to OpenShift
+You can deploy to OpenShift for development. This process will use your github repo to launch the website,
```

Comment:
Same thing, clarify that this is on your own instance of OpenShift cluster. Does this matter if the cluster is 3.x or 4.x?

Reply:
No difference on v3 vs v4 deployment